source/tutorials/moving-pictures.rst

```diff
@@ -367,7 +367,7 @@ Next, we can view the taxonomic composition of our samples with interactive bar
 Differential abundance analysis
 -------------------------------
 
-Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. We work around this by adding a pseudocount of 1 to every count in our ``FeatureTable[Frequency]`` table. We can run this on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
+Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. In order to use ANCOM, a ``FeatureTable[Composition]``artifact must be built using frequency data. To do this, a ``FeatureTable[Frequency]`` artifact containing the relative frequencies must be used as input to add-pseudocount, which will then output the ``FeatureTable[Composition]`` artifact necessary to use ANCOM. We can run this on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
```

Comment:
`...``FeatureTable[Composition]``artifact must be b...` -> there's a missing space between the backtick and artifact which will cause issues.

`FeatureTable[Composition]` is not strictly based on relative frequencies. Relative frequencies are also not the input to `add-pseudocount`.

We can brainstorm on some better language in-person if you'd like.